integration-tests/src/test/java/eu/tsystems/mms/tic/testframework/test/guielement/AbstractGuiElementStandardFunctionsTest.java

```diff
@@ -172,7 +171,6 @@ public void testT12N_GuiElement_assertIsSelected() {
      * Test if GuiElement.asserts().assertIsNotSelectable works for an unselectable element
      */
     @Test
-    @Fails(validFor = "unsupportedBrowser=true", description = "Does not work in this browser!")
```

Comment:
Do we officially drop a supported browser in this PR (necessitating a Major version increase and I'd expect
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

to be set in the description) or have we done so in the past and this annotation was missed on removal?

Reply:
This is not a breaking change but only a cleanup of outdated stuff. Some years ago the tests were executed in PhantomJS. And some of this tests was not working.
Now we are using Chrome.